File: BearAndBigBrother.cpp
#include <iostream>
using namespace std;

int countYears(int weightOfLimak, int weightOfBob) {
    int requiredYears = 0;
    while (weightOfLimak <= weightOfBob) {
        weightOfLimak = weightOfLimak * 3;
        weightOfBob = weightOfBob * 2;
        requiredYears += 1;
    }
    return requiredYears;
}

int main() {
    int weightOfLimak, weightOfBob, requiredYears;
    cin >> weightOfLimak >> weightOfBob;
    requiredYears = countYears(weightOfLimak, weightOfBob);
    cout<< requiredYears <<endl;
    return 0;
}

File: bear_and_big_brother.js
function countYears(weightOfLimak, weightOfBob) {
    var requiredYears = 0;
    while(weightOfLimak <= weightOfBob) {
        weightOfLimak = weightOfLimak * 3;
        weightOfBob = weightOfBob * 2;
        requiredYears += 1;
    }
    return requiredYears;
}

var inputLine = readline().split(' ').map(x => Number.parseInt(x));
var weightOfLimak = inputLine[0];
var weightOfBob = inputLine[1];
var requiredYears = countYears(weightOfLimak, weightOfBob);
print(requiredYears);
